src/manager.js

```diff
@@ -696,11 +697,10 @@ AdManager.prototype.refreshSlot = function (domElement) {
   }
 
   var slot = this.slots[domElement.id];
-  var ads = this.findAds(domElement);
 
   if (slot) {
     domElement.setAttribute('data-ad-load-state', 'loading');
-    this.refreshSlots([slot], ads);
```

Comment:
`ads` was still being passed in, even though it's not even used in the `refreshSlots` method